modules/compatibility/src/test/java/org/apache/ignite/compatibility/persistence/IgnitePKIndexesMigrationToUnwrapPkTest.java

```diff
@@ -248,14 +249,20 @@ private static void assertDontUsingPkIndex(IgniteEx igniteEx, String tblName) {
 
             cfg.setPeerClassLoadingEnabled(false);
 
-            DataStorageConfiguration memCfg = new DataStorageConfiguration()
+            DataStorageConfiguration storageCfg = new DataStorageConfiguration()
                 .setDefaultDataRegionConfiguration(
-                    new DataRegionConfiguration().setPersistenceEnabled(true)
-                        .setInitialSize(1024 * 1024 * 10).setMaxSize(1024 * 1024 * 15))
-                .setSystemRegionInitialSize(1024 * 1024 * 10)
-                .setSystemRegionMaxSize(1024 * 1024 * 15);
-
-            cfg.setDataStorageConfiguration(memCfg);
+                    new DataRegionConfiguration()
+                            .setPersistenceEnabled(true)
+                            .setInitialSize(1024 * 1024 * 10)
+                            .setMaxSize(1024 * 1024 * 15)
+                )
+                .setSystemDataRegionConfiguration(
+                    new SystemDataRegionConfiguration()
+                            .setInitialSize(10 * 1024 * 1024)
```

Comment:
multipliers in asc order, but for previous region is in desk - i'm asking make it the same way, holnig my monocle

Reply:
Will fix it.